tecton_client/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.2.1+local"
```

Comment:
Do you want this to say local?

Reply:
Yes, it's removed in the cut pipeline, it's an error that it's not here currently.